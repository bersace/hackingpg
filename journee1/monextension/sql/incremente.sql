CREATE EXTENSION monextension;
SELECT incremente(10);
