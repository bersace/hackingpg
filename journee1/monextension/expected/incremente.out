CREATE EXTENSION monextension;
SELECT incremente(10);
 incremente 
------------
         11
(1 row)

